### DISCUSSION TITLE
Filament Select: maxItems not working with callback

### DISCUSSION THREAD

**Discussion by iftekhar-ifty** [filamentphp/filament#9288] [General]
```
return $form
    ->schema([
        TextInput::make('max_items_input')
            ->live(onBlur: true),
        
        Select::make('technologies')
            ->multiple()
            ->options([
                'tailwind' => 'Tailwind CSS',
                'alpine' => 'Alpine.js',
                'laravel' => 'Laravel',
                'livewire' => 'Laravel Livewire',
            ])
            ->maxItems(fn (Get $get): string => $get('max_items_input') ?? 1) // It doesn't work.
            ->helperText(fn (Get $get): string => $get('max_items_input') ?? 1) // It works.

    ]);
```

**Comment by leandrocfe** [comment]
Convert to int?

**Reply by iftekhar-ifty** [reply]
i already nothing happened. inside  ->maxItems(fn (Get $get): string => $get('max_items_input') ?? 1) not reactive but when i try is same thing on helperText its working

**Comment by sinnbeck** [comment]
How doesnt it work? You get an error? Or it allows too many? Or too few?

Edit: It seems that the plugin isnt reinitialized when that changes. 
![image](https://github.com/filamentphp/filament/assets/13980140/0c73400a-6305-4e66-9991-e334284e0018)
So you will need to find a way to reload the component script

**Reply by iftekhar-ifty** [reply]
yeah i am not getting any error it just take 1 from here "fn (Get $get): string => $get('max_items_input') ?? 1" inside this function not reactive

**Reply by sinnbeck** [reply]
Yeah it never reinitializes the js library. I don't think it's possible to force it currently (at least I haven't found a way to)

**Reply by iftekhar-ifty** [reply]
maybe
